README.md

```diff
@@ -440,4 +440,14 @@ export NUM_EXTRA_WORKERS=2
 ```
 
 After initial deployment, a file containing the BareMetalHost manifests can be
-applied, e.g `oc apply -f ocp/ostest/extra_host_manifests.yaml`
+applied:
+
+```
+oc apply -f ocp/ostest/extra_host_manifests.yaml
+```
+
+Once completed, it's possibile to scale up the machineset to provision the extra workers.
+
+```
+oc scale $(oc get machineset -o name) --replicas=4 -n openshift-machine-api
```

Comment:
This will fail in a non-obvious way if the user isn't in the openshift-machine-api namespace (because the subshell oc get doesn't specify `-n`)

Would it be clearer to just show both commands, e.g

```
$ oc get machineset -n openshift-machine-api
NAME              DESIRED   CURRENT   READY   AVAILABLE   AGE
ostest-worker-0   2         2         2       2           27h
$ oc scale machineset ostest-worker-0 --replicas=3 -n openshift-machine-api
machineset.machine.openshift.io/ostest-worker-0 scaled
```

Reply:
Yeah looks better